tmpi

```diff
@@ -55,15 +63,23 @@ if [[ ${1} == runmpi ]] ; then
     # we are being started as one of many processes by mpirun.
     shift
 
+    if [ -n "${mpich}" ]; then
+        mpi_rank=${PMI_ID}
+        mpi_env="-e MPI -e HYDRA -e PMI"
```

Comment:
I suggest changing `-e HYDRA` to `-e HYD` but otherwise this looks good.

Reply:
Fixed.